TicTacToe.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+
+class TicTacToe {
+    public:
+        TicTacToe() {
+            for (int i = 0; i < 9; i++) {
+                arr_[i / 3][i % 3] = '1' + i;
```

Comment:
Not sure what is going on in this loop, maybe add comments to clarify logic. Also, I think you might be trying to set each element of the matrix to a char + int which might result in a typing error.